docs/source/privacy/index.md

```diff
@@ -0,0 +1,23 @@
+---
+title: "Data Collection"
+sidebar_title: "Data Collection"
+description: "Information about data collected by Rover"
+---
+
+By default, Rover collects some anonymous usage data to help better understand the needs of our users. 
+
+## Collected Data
+
+Rover reports a few different pieces of information each time you run a command:
+
+- The command that was run _(excluding any identifiable arguments such as file-system paths or profile names)_.
```

Comment:
Remove ending punctuation for each list item (include only if an item is a complete sentence)